libs/utils/android/system.py

```diff
@@ -37,4 +37,48 @@ def set_airplane_mode(target, on=True):
                        '--ez state {}'\
                        .format(ap_state))
 
+    @staticmethod
+    def start(target, apk_name, activity_name):
+        """
+        Start an application.
+
+        :param apk_name: name of the apk
+        :type apk_name: str
+
+        :param activity_name: name of the activity to launch
+        :type activity_name: str
+        """
+        target.execute('am start -n {}/{}'.format(apk_name, activity_name))
+
+    @staticmethod
+    def force_stop(target, apk_name, clear=False):
+        """
+        Stop the application and clear its data if necessary.
+
+        :param target: instance of devlib Android target
+        :type target: devlib.target.AndroidTarget
+
+        :param apk_name: name of the apk
+        :type apk_name: str
+
+        :param clear: clear application data
+        :type clear: bool
+        """
+        target.execute('am force-stop {}'.format(apk_name))
+        if clear:
+            target.execute('pm clear {}'.format(apk_name))
+
+    @staticmethod
+    def tap(target, x, y):
```

Comment:
We should add a percentage version of this call, where we can exploit another (memoaized) screen size to compute the actual coords from a screen percentage coordinate.
Perhaps such a method is good enough to be quite portable... but let's do it on a second time.